cls/SourceControl/Git/Utils.cls

```diff
@@ -404,6 +420,7 @@ ClassMethod Sync(Msg As %String) As %Status
         do ..Fetch()
         do ..Pull()
         do ..SyncCommit(Msg)
+        do ..MergeDefaultRemoteBranch()
```

Comment:
I think we should Push before attempting to merge (as well as after). That way if there's a conflict it can be resolved in the remote with the latest from the feature branch already available there. A subsequent Pull after such a resolution should be able to get things back to a consistent state.